target-platform/p2-repo-equinox_3.11.1/pom.xml

```diff
@@ -0,0 +1,168 @@
+                  <goals>
+                     <goal>read-project-properties</goal>
+                  </goals>
+                  <configuration>
+                     <files>
+                        <file>${basedir}/../config/kura.target-platform.build.properties</file>
+                     </files>
+                  </configuration>
+               </execution>
+            </executions>
+         </plugin>
+			<plugin>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>copy-bundles-for-publishing</id>
+						<phase>process-resources</phase>
+						<goals>
+							<goal>copy</goal>
+						</goals>
+						<configuration>
+							<artifactItems>
+                                <artifactItem>
+                                    <groupId>org.apache.felix</groupId>
+                                    <artifactId>org.apache.felix.dependencymanager</artifactId>
+                                    <version>${org.apache.felix.dependencymanager.version}</version>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.apache.felix</groupId>
+                                    <artifactId>org.apache.felix.deploymentadmin</artifactId>
+                                    <version>${org.apache.felix.deploymentadmin.version}</version>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.knowhowlab</groupId>
+                                    <artifactId>org.knowhowlab.osgi.monitoradmin</artifactId>
+                                    <version>${org.knowhowlab.osgi.monitoradmin.version}</version>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>osgi</groupId>
+                                    <artifactId>osgi.cmpn</artifactId>
+                                    <version>${osgi.cmpn.version}</version>
+                                </artifactItem>
+							</artifactItems>
+							<stripVersion>true</stripVersion>
+							<outputDirectory>${project.basedir}/target/source/plugins</outputDirectory>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.sonatype.tycho</groupId>
+				<artifactId>tycho-p2-extras-plugin</artifactId>
+				<version>0.11.0</version>
+				<executions>
+					<execution>
+						<phase>prepare-package</phase>
+						<goals>
+							<goal>publish-features-and-bundles</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<compress>false</compress>
+					<!-- additional arguments example <additionalArgs>-configs win32.win32.x86</additionalArgs> -->
+				</configuration>
+			</plugin>
+			<plugin>
+				<artifactId>maven-antrun-plugin</artifactId>
+                <version>1.8</version>
+				<executions>
```

Comment:
There is none. As this always messes up files. I wish those files would have a common ways of dealing with this. If you enable "special characters view" you can see the mess these files are in.

@MMaiero  @cdealti  So if this is the only problem I would like to go ahead and merge this